torchgen/gen.py

```diff
@@ -152,6 +156,152 @@ def construct_mapping(self, node, deep=False):  # type: ignore[no-untyped-def]
 ParsedYaml = namedtuple("ParsedYaml", ["native_functions", "backend_indices"])
 
 
+def pre_group_native_functions(
```

Comment:
comment?

Reply:
This was pre-existing and I moved it further up in the file, but you're right it deserves a comment either way